tex/gregoriotex.lua

```diff
@@ -830,8 +804,6 @@ local function post_linebreak(h, groupcode, glyphes)
           new_score_last_syllables[syl_id] = syl_id
         end
       end
-      -- we reinitialize the shift value, because it may change according to the line
-      currentshift=0
```

Comment:
This was the bug: currentshift should have been reset at every syllable, like currentfont is.